apps/climatemappedafrica/src/components/Card/Content.js

```diff
@@ -44,13 +45,22 @@ function Content({
       <RichTypography variant="h5" {...titleProps} className={classes.title}>
         {title}
       </RichTypography>
-      <RichTypography
-        variant="subtitle2"
-        {...descriptionProps}
-        className={classes.description}
-      >
-        {description}
-      </RichTypography>
+      {/* Support for rich text while keeping backwards compatibility */}
+      {Array.isArray(description) ? (
```

Comment:
Cool but lets add a story to track this... ideally we should support rich text everywhere.

Reply:
Tracking This in #962